book/setting_up.md

```diff
@@ -329,11 +329,10 @@ void hello_world();
 void MainWindow::onClick() { 
     // Call the `hello_world` function to print a message to stdout
     hello_world(); 
-    }
+}
 
 MainWindow::MainWindow(QWidget *parent) : QMainWindow(parent) {
   button = new QPushButton("Click Me", this);
-  button->show();
```

Comment:
Just out of curiosity, why do we not need to call `button->show()` here?

Reply:
The button is made a child of the mainwindow instance in the widget hierarchy. And when the mainwindow instance is asked to show(), it will by default also make all the child widgets shown, unless they had been explicitly set to be hidden before.
Cmp. 
http://doc.qt.io/qt-5/qwidget.html#show
http://doc.qt.io/qt-5/qwidget.html#isHidden